.github/workflows/upload-test-stats.yml

```diff
@@ -58,6 +58,23 @@ jobs:
           python3 -m tools.stats.upload_test_stats --workflow-run-id "${WORKFLOW_RUN_ID}" --workflow-run-attempt "${WORKFLOW_RUN_ATTEMPT}" --head-branch "${HEAD_BRANCH}"
           python3 -m tools.stats.upload_sccache_stats --workflow-run-id "${WORKFLOW_RUN_ID}" --workflow-run-attempt "${WORKFLOW_RUN_ATTEMPT}"
 
+      - name: Upload test artifacts
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          WORKFLOW_ARTIFACTS_URL: ${{ github.event.workflow_run.artifacts_url }}
+          WORKFLOW_RUN_ID: ${{ github.event.workflow_run.id }}
+          WORKFLOW_RUN_ATTEMPT: ${{ github.event.workflow_run.run_attempt }}
+          REPO_FULLNAME: ${{ github.event.workflow_run.repository.full_name }}
+        run: |
+          echo "${WORKFLOW_ARTIFACTS_URL}"
+
+          # Check if the file exists before running it, this is to avoid breaking PRs that are not yet rebasing on top of this change
+          if [[ -f "tools/stats/upload_artifacts.py" ]]; then
+            # Note that in the case of Linux and Windows, their artifacts have already been uploaded to S3, so there simeply won't be
```

Comment:
typo for simeply

I'm don't think that checking if the file exists is required.  This workflow is run on workflow_run completed trigger, which uses the master version of the workflow file, and I assume the master version of the repo?

Reply:
I see, Let me double check this and get rid of the check if that's the case (running on master version)